master/internal/api_model.go

```diff
@@ -173,7 +174,8 @@ func (a *apiServer) PatchModel(
 		log.Infof("model (%d) name changing from \"%s\" to \"%s\"",
 			currModel.Id, currModel.Name, req.Model.Name.Value)
 		if err = a.clearModelName(ctx, req.Model.Name.Value); err != nil {
-			return nil, errors.Wrap(err, "error renaming model")
+			return nil, status.Errorf(codes.AlreadyExists,
```

Comment:
I guess here it is the same case as above. `codes.InvalidArgument` is more appropriate